apps/intellij/src/main/kotlin/dev/nx/console/generate/actions/NxReMoveProjectActions.kt

```diff
@@ -54,6 +54,13 @@ open class NxReMoveProjectActionBase(val mode: String) : AnAction() {
 
         val generators = nxlsService.generators()
         val moveGenerators = generators.map { it.name }.filter { it.contains(Regex(":${mode}$")) }
+
+        if (moveGenerators.isEmpty()) {
+            throw Exception(
+                "No $mode generators found. Make sure the nxls is configured correctly."
```

Comment:
```suggestion
                "No $mode generators found. Make sure that node_modules are installed, or set the root of Nx Console to point to a Nx workspace in editor settings."
```

Reply:
Maybe show error notification with two possible actions Run `pakageName` install, or show Nx settings to fix path ...

Reply:
Yes we will look into setting up better error handling in the future. For now I just want to avoid a nondescriptive 'index out of bounds' error